callerinfo.go

```diff
@@ -0,0 +1,42 @@
+package log
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+const (
+	UnknownFile = "unknown_file"
+	UnknownPath = "unknown_path"
+	UnknownFunc = "unknown_func"
```

Comment:
These don't need to be exported. I wouldn't even pull them out as constants, since they're only used once and their meaning is obvious.